akka-docs-dev/rst/java/stream-testkit.rst

```diff
@@ -0,0 +1,25 @@
+.. _stream-testkit-java:
+
+###############
+Testing streams
+###############
+
+Akka Streams comes with an :mod:`akka-stream-testkit` module that provides tools which can be used for controlling and asserting various parts of the stream pipeline.
+
+Probe Sink
+==========
+
+Using probe as a `Sink` allows manual control over demand and assertions over elements coming downstream. Streams testkit provides a sink that materializes to a :class:`TestSubscriber.Probe`.
+
+.. includecode:: ../../../akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/StreamTestKitDocTest.java#test-sink-probe
+
+Probe Source
+============
+
```

Comment:
it should mention that most tests can use a strict collection `Source(iterable)` as source